### PR TITLE
VIH-10687 Add visual cue for IOS users

### PR DESCRIPTION
**Pull request by oliver-scott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/VIH-10687

### Change description ###

- Display a confirmation popup to iPhone and iPad users (participants and johs) when joining the waiting room
- Move `yes-no-popup-base` component into shared folder
- Remove safari browser check from `isIpad()`

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/services/device-detection.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Logger } from 'src/app/services/logging/logger-base';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DeviceDetectionService {
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/device-detection.service.ts:7]: there should already be a device detector service (think it's called `DeviceTypeService`) that wraps the device type library. might want to check that out

**Review by shaed-parkar — COMMENTED** [review]
Looks good but I think you may have reinvented the wheel here a bit

File: VideoWeb/VideoWeb/ClientApp/src/app/services/device-detection.service.ts (hunk 1 above)
Comment (oliver-scott) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/device-detection.service.ts]: thanks. Have changed the implementation to use that service instead, and removed the new one

**Review by shaed-parkar — APPROVED** [review]
Looks good, let's hope that sorts this long term issue out!

**Review by komaldabhi1509 — APPROVED** [review]

**Review by SwapnaVersion1 — APPROVED** [review]

**Pull request merged**